python/ray/autoscaler/kubernetes/operator_configs/operator_config.yaml

```diff
@@ -0,0 +1,53 @@
+operator_role:
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ray-operator-serviceaccount
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ray-operator-role
+rules:
+- apiGroups: ["", "rbac.authorization.k8s.io"]
+  resources: ["configmaps", "pods", "pods/exec", "services", "serviceaccounts", "roles", "rolebindings"]
+  verbs: ["get", "watch", "list", "create", "delete", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ray-operator-rolebinding
+subjects:
+- kind: ServiceAccount
+  name: ray-operator-serviceaccount
+roleRef:
+  kind: Role
+  name: ray-operator-role
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ray-operator-pod
+spec:
+  serviceAccountName: ray-operator-serviceaccount
+  containers:
+  - name: ray
+    imagePullPolicy: Always
+    image: rayproject/ray:nightly
+    command: ["/bin/bash", "-c", "--"]
+    # TODO Install a proper entry point for the operator, instead of using the path to operator.py.
+    args: ["python $OPERATOR_PATH; trap : TERM INT; sleep infinity & wait;"]
```

Comment:
do we need the sleep infinity? why can't operator.py just run forever?

Reply:
> Wow, this is very simple! Looks good for a first pass.

Yeah -- I figured it would be prudent to use the existing ray code as much as possible. I'd like to continue doing that. The ray autoscaler is, after all, an operator which orchestrates containers running ray processes...

Reply:
> do we need the sleep infinity? why can't operator.py just run forever?

You're right. But for some reason, when I ran it without the `sleep infinity`, the operator pod entered completed status upon startup. It probably has something to do with the particulars of the subprocess logic in  `ray._private.services.start_monitor()`.

Reply:
I'll look into why the Pod is getting to completed status.
For now, the `sleep infinity` is also helpful for debugging.

Reply:
Sounds good. Not a blocker, we can merge as-is -- I just want to make sure we understand the behavior.